packages/request-node/src/request/persistTransaction.ts

```diff
@@ -83,12 +86,27 @@ export default class PersistTransactionHandler {
           )}`);
       });
 
-      this.logger.debug(`persistTransaction successfully completed`, ['metric', 'successRate']);
+      this.logger.debug(
+        `persistTransaction successfully completed ${JSON.stringify({
+          transactionHash,
+          channelId: clientRequest.body.channelId,
+          topics: clientRequest.body.topics,
+          transactionData: clientRequest.body.transactionData,
+        })}`,
+        ['metric', 'successRate'],
+      );
```

Comment:
Please remove the topics and transactionData from this log statement.
```suggestion
      this.logger.debug(
        `persistTransaction successfully completed ${JSON.stringify({
          transactionHash,
          channelId: clientRequest.body.channelId,
        })}`,
        ['metric', 'successRate'],
      );
```